lib/medium-admin.php

```diff
@@ -136,10 +136,16 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters('medium_allowed_post_types', array('post'));
```

Comment:
Of course, it won't be sent if status isn't set to "Draft, Public, or Unlisted", which can only be the case on the post screen, because it's the only place that sends that field in the request.